contribute.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "DXR",
+    "description": "An intelligent source code browser.",
+    "repository": {
+        "url": "https://github.com/mozilla/dxr",
+        "license": "MIT",
+        "tests": "https://ci.mozilla.org/job/dxr/"
+    },
+    "participate": {
+        "home": "https://dxr.readthedocs.org/en/latest/community.html",
+        "docs": "https://dxr.readthedocs.org/en/latest/development.html",
```

Comment:
Should this be https://dxr.readthedocs.org/en/latest/index.html, or is the semantic "developer docs"?